modules/dedicated-creating-your-cluster.adoc

```diff
@@ -39,4 +39,35 @@ To access your {product-title} cluster:
 . From link:https://cloud.redhat.com/openshift[cloud.redhat.com/openshift], click
  on the cluster you want to access.
 
- . Click *Launch Console*.
+ . Click *Open Console*.
+
+= Configuring your identity providers
+
+To configure identity providers for your {product-title} cluster:
+
+. Navigate to the *Cluster* page and *Access control* tab.
+
+. Click the *Add identity provider* button.
```

Comment:
Here and elsewhere in the procedure, you can shorten to just, `Click *Add identity provider*`. Most of our docs don't indicate that it's a button, following IBM Style Guide: 

> Refer to a button by its label only or tooltip name only; do not use _button_ or _push button_.

Reply:
I've definitely had this on several occasions. 😁